php/commands/super-admin.php

```diff
@@ -27,9 +27,10 @@ public function _list( $_, $assoc_args ) {
 	 * <user>...
 	 * : One or more user IDs, user emails, or user logins.
 	 */
-	public function add( $args, $_ ) {
+	public function add() {
+		$request = \WP_CLI\Request::get_from_argv();
 
-		$users = $this->fetcher->get_many( $args );
+		$users = $this->fetcher->get_many( $request->get_positional_args() );
```

Comment:
Potentially, `get_positional_args()` could return null until the command is known. Or, we could unset the positional args set to the command.